pkg/anago/release_test.go

```diff
@@ -144,3 +144,12 @@ func TestPrepareWorkspaceRelease(t *testing.T) {
 		}
 	}
 }
+
+func TestCreateAnnouncement(t *testing.T) {
+	opts := anago.DefaultReleaseOptions()
+	sut := anago.NewDefaultRelease(opts)
+	mock := &anagofakes.FakeReleaseImpl{}
+	sut.SetClient(mock)
+	err := sut.CreateAnnouncement(git.DefaultBranch)
+	require.Nil(t, err)
+}
```

Comment:
This test needs now an additional case if the announcement generation fails.

Reply:
All tests should be now complete